Libraries/acdriver/Sources/CompileAction.cpp

```diff
@@ -27,10 +62,303 @@ CompileAction::
 {
 }
 
+static bool
+CompileAsset(
+    std::shared_ptr<xcassets::Asset::Catalog> const &catalog,
+    Options const &options,
+    CompileOutput *compileOutput,
```

Comment:
Are `result` and `compileOutput` considered as out parameters here?  If so, maybe they ought to move behind `parent` and `asset`.